.circleci/config.yml.in

```diff
@@ -642,19 +642,25 @@ jobs:
   build_docs:
     <<: *binary_common
     docker:
-      - image: "pytorch/manylinux-cuda100"
+      - image: "python:<< parameters.python_version >>"
```

Comment:
just checking, the `python:3.8` in the PR description should actually be `python:3.x` right?

Reply:
Yes, that's correct. Updated the description.